### PR TITLE
Swift 2.2

### PR DESCRIPTION
**Pull request by ikesyo**
We would need updating Argo and ReactiveCocoa as well.
- [x] Argo: [v3.0](https://github.com/thoughtbot/Argo/releases/tag/v3.0.0)
- [x] ReactiveCocoa: [v4.1](https://github.com/ReactiveCocoa/ReactiveCocoa/releases/tag/v4.1.0)

### CODE REVIEW

**Comment by ikesyo** [comment]
ff2fde2: See https://github.com/facebook/xctool/issues/666.

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 language: objective-c
 notifications:
   email: false
-osx_image: xcode7.2
+osx_image: xcode7.3
 before_install:
-  - brew update
-  - brew upgrade carthage
```
Comment (mdiep) [inline .travis.yml:7]: I think we should keep this so that Tentacle automatically uses the latest Carthage release.

**Comment by mdiep** [comment]
Thanks for working on this!

File: .travis.yml (hunk 1 above)
Comment (ikesyo) [inline .travis.yml:7]: Addressed by 95e4ec8. :zap: Otherwise, that would fail with the error: https://travis-ci.org/mdiep/Tentacle/jobs/118976586#L364

**Comment by ikesyo** [comment]
@mdiep Argo 2.3.0 is buildable with Swift 2.2. Should we wait for a new release (maybe 3.0, see https://github.com/thoughtbot/Argo/pull/333#issuecomment-204719852) or omit it for a quick release?

**Comment by mdiep** [comment]
Let's wait for the 3.0 release.

**Comment by ikesyo** [comment]
Argo is finally updated. :sparkles:

**Comment by mdiep** [comment]
Thanks! 💖

**Pull request merged**